node/Cargo.toml

```diff
@@ -12,50 +12,90 @@ edition = "2018"
-sp-consensus = { git = "https://github.com/paritytech/substrate", package = "sp-consensus" }
-sc-consensus = { git = "https://github.com/paritytech/substrate", package = "sc-consensus" }
-sc-finality-grandpa = { git = "https://github.com/paritytech/substrate", package = "sc-finality-grandpa" }
-sp-finality-grandpa = { git = "https://github.com/paritytech/substrate", package = "sp-finality-grandpa" }
-sc-client-api = { git = "https://github.com/paritytech/substrate", package = "sc-client-api" }
-sp-runtime = { git = "https://github.com/paritytech/substrate", package = "sp-runtime" }
-sp-timestamp = { git = "https://github.com/paritytech/substrate", package = "sp-timestamp" }
-
-# These dependencies are used for the node's RPCs
-jsonrpc-core = "18.0.0"
-sc-rpc = { git = "https://github.com/paritytech/substrate", package = "sc-rpc" }
-sp-api = { git = "https://github.com/paritytech/substrate", package = "sp-api" }
-sc-rpc-api = { git = "https://github.com/paritytech/substrate", package = "sc-rpc-api" }
-sp-blockchain = { git = "https://github.com/paritytech/substrate", package = "sp-blockchain" }
-sp-block-builder = { git = "https://github.com/paritytech/substrate", package = "sp-block-builder" }
-sc-basic-authorship = { git = "https://github.com/paritytech/substrate", package = "sc-basic-authorship" }
-substrate-frame-rpc-system = { git = "https://github.com/paritytech/substrate", package = "substrate-frame-rpc-system" }
-pallet-transaction-payment-rpc = { git = "https://github.com/paritytech/substrate", package = "pallet-transaction-payment-rpc" }
-pallet-contracts-rpc = { git = "https://github.com/paritytech/substrate", package = "pallet-contracts-rpc" }
-
-# These dependencies are used for runtime benchmarking
-frame-benchmarking = { git = "https://github.com/paritytech/substrate", package = "frame-benchmarking" }
-frame-benchmarking-cli = { git = "https://github.com/paritytech/substrate", package = "frame-benchmarking-cli" }
-
-canvas-runtime = { version = "0.1.0", path = "../runtime" }
-
 [build-dependencies]
-substrate-build-script-utils = { git = "https://github.com/paritytech/substrate", package = "substrate-build-script-utils" }
+substrate-build-script-utils = { git = 'https://github.com/paritytech/substrate', branch = "cmichi-tmp-experiment" }
 
 [[bin]]
-name = "canvas"
-path = "src/main.rs"
+name = 'canvas-node'
+
+[features]
+default = []
+runtime-benchmarks = ['canvas-runtime/runtime-benchmarks']
+
+[dependencies]
+derive_more = '0.99.2'
+log = '0.4.14'
+codec = { package = 'parity-scale-codec', version = '2.0.0' }
+structopt = '0.3.8'
+serde = { version = '1.0.119', features = ['derive'] }
+hex-literal = '0.3.1'
+
+# RPC related Dependencies
+jsonrpc-core = '15.1.0'
+
+# Local Dependencies
+canvas-runtime = { path = '../runtime' }
+
+# Substrate Dependencies
+frame-benchmarking = { git = 'https://github.com/paritytech/substrate', branch = "cmichi-tmp-experiment" }
```

Comment:
These branches could do with more descriptive names?

Reply:
Good point, I've just created [cmichi-contract-experiment-patched-v0.9.9](https://github.com/paritytech/substrate/tree/cmichi-contract-experiment-patched-v0.9.9) which contains the same commits, so we can just replace the name with this one.

Reply:
I ended up making [hc-contract-experiment-patched-v0.9.9](https://github.com/paritytech/substrate/tree/hc-contract-experiment-patched-v0.9.9) instead. Now all the branches across the different repos have the same name, which makes changes slightly more straightforward